tests/src/Base/UniqueNameManager.cpp

```diff
@@ -0,0 +1,104 @@
+/***************************************************************************
+ *   Copyright (c) 2024 Kevin Martin <kpmartin@papertrail.ca>              *
+ *                                                                         *
+ *   This file is part of the FreeCAD CAx development system.              *
+ *                                                                         *
+ *   This library is free software; you can redistribute it and/or         *
+ *   modify it under the terms of the GNU Library General Public           *
+ *   License as published by the Free Software Foundation; either          *
+ *   version 2 of the License, or (at your option) any later version.      *
+ *                                                                         *
+ *   This library  is distributed in the hope that it will be useful,      *
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
+ *   GNU Library General Public License for more details.                  *
+ *                                                                         *
+ *   You should have received a copy of the GNU Library General Public     *
+ *   License along with this library; see the file COPYING.LIB. If not,    *
+ *   write to the Free Software Foundation, Inc., 59 Temple Place,         *
+ *   Suite 330, Boston, MA  02111-1307, USA                                *
+ *                                                                         *
+ ***************************************************************************/
+#include <gtest/gtest.h>
+#include <string>
+#include <Base/UniqueNameManager.h>
+
+// NOLINTBEGIN(cppcoreguidelines-*,readability-*)
+TEST(UniqueNameManager, TestUniqueName1)
+{
+    EXPECT_EQ(Base::UniqueNameManager().makeUniqueName("Body"), "Body");
+}
+
+TEST(UniqueNameManager, TestUniqueName2)
```

Comment:
Generally speaking I prefer longer and more explicit test names, rather than numbers, so when they fail in the CI it's clearer what the problem is.